src/cplscheme/ParallelCouplingScheme.cpp

```diff
@@ -37,22 +37,43 @@ void ParallelCouplingScheme::exchangeFirstData()
 
 void ParallelCouplingScheme::exchangeSecondData()
 {
+  // @todo bundle receiveConvergence, specific moveToNextWindow and sendData into one function?
```

Comment:
```suggestion
```
IMO no. Would only make things harder to read. More occasions below.

Reply:
Ok. Removed the comments in 1934853707403ada41aa9c8b838351ecf883fb89